<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDFH</class>
 <widget class="QWidget" name="OptionsDFH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Damage Factor History Report Options</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="mButtonHPMDR">
        <property name="text">
         <string>High Power Microwave (HPM) Damage Report</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mButtonPDR">
        <property name="text">
         <string>Partial Damage Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mButtonDEDR">
        <property name="text">
         <string>Directed Energy (DE) Damage Report</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="directedEnergyLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="mCheckBoxDEAH">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DE Action History</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="mCheckBoxActingIFF">
          <property name="text">
           <string>Acting Platform's IFF</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="mCheckBoxAgainstIFF">
          <property name="text">
           <string>Against Platforms's IFF</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="mCheckBoxLP">
          <property name="text">
           <string>Launching Platform</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="mCheckBoxTST">
          <property name="text">
           <string>Total Slew Time</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="mCheckBoxDT">
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="mCheckBoxMKT">
          <property name="text">
           <string>Maximum Kill Time</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="mCheckBoxLF">
          <property name="text">
           <string>Laser Fuel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="mCheckBoxDPk">
          <property name="text">
           <string>Desired Pk</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="mCheckBoxPA">
          <property name="text">
           <string>Planned Accumulation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="mCheckBoxPKM">
          <property name="text">
           <string>Planned Kill Metric</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="mCheckBoxPBS">
          <property name="text">
           <string>Planned Beam Spread</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="mCheckBoxPPI">
          <property name="text">
           <string>Planned Peak Intensity</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="mCheckBoxSS">
          <property name="text">
           <string>Spot Size</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QCheckBox" name="mCheckBoxA">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QCheckBox" name="mCheckBoxAP">
          <property name="text">
           <string>Aim Point</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="mCheckBoxAPk">
          <property name="text">
           <string>Actual Pk</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="mCheckBoxAA">
          <property name="text">
           <string>Actual Accumulation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="mCheckBoxAKM">
          <property name="text">
           <string>Actual Kill Metric</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="mCheckBoxABS">
          <property name="text">
           <string>Actual Beam Spread</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="mCheckBoxAPI">
          <property name="text">
           <string>Actual Peak Intensity</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QCheckBox" name="mCheckBoxPC">
          <property name="text">
           <string>Power Correction</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QCheckBox" name="mCheckBoxWPC">
          <property name="text">
           <string>Wavefront Power Correction</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QCheckBox" name="mCheckBoxBDRD">
          <property name="text">
           <string>Beam Delivery Random Draw</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QCheckBox" name="mCheckBoxBDP">
          <property name="text">
           <string>Beam Delivery Probability</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Report Type</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
